about.html

```diff
@@ -37,117 +34,122 @@
+            <div class="last-meet">
+              <div class="image-container">
+                <img src="./img/venezuela-2022.jpg" alt="Venezuela 2022" />
+                <div class="text-overlay">
+                  <h3><span>2022</span> WFC in Venezuela</h3>
+                </div>
+              </div>
+              <div class="image-container">
+                <img src="./img/brasil-2021.jpg" alt="Brasil 2021" />
+                <div class="text-overlay">
+                  <h3><span>2021</span> WFC in Goias</h3>
+                </div>
               </div>
             </div>
           </div>
         </div>
-      </div>
-    </section>
+      </section>
 
-    <section class="main-partners">
-      <h2 class="text-center">Partners</h2>
-      <div class="container">
-        <div class="row">
-          <div class="col d-flex justify-content-center sp">
-            <img src="./iconos/mozilla.svg" alt="mozilla" />
-            <img src="./iconos/google.svg" alt="google" />
-            <img src="./iconos/naver.svg" alt="naver" />
-            <img src="./iconos/daum.svg" alt="daum" />
-            <img src="./iconos/airbnb.svg" alt="airbnb" />
+      <section class="main-partners">
+        <div class="container">
+          <div class="row">
+            <div class="col">
+              <h2 class="text-center mb-4">Partners</h2>
+            </div>
+          </div>
+          <div class="row justify-content-center">
+            <div class="col-4 col-md-2 d-flex align-items-center">
+              <img src="./img/iconos/mozilla.svg" alt="mozilla" />
+            </div>
+            <div class="col-4 col-md-2 d-flex align-items-center">
+              <img src="./img/iconos/google.svg" alt="google" />
+            </div>
+            <div class="col-4 col-md-2 d-flex align-items-center">
+              <img src="./img/iconos/naver.svg" alt="naver" />
+            </div>
+            <div class="col-4 col-md-2 d-flex align-items-center">
+              <img src="./img/iconos/daum.svg" alt="daum" />
+            </div>
+            <div class="col-4 col-md-2 d-flex align-items-center">
+              <img src="./img/iconos/airbnb.svg" alt="airbnb" />
+            </div>
           </div>
         </div>
-      </div>
-    </section>
-    <footer class="main-footer">
+      </section>
+    </main>
+    <footer class="main-footer py-3">
       <div class="container">
         <div class="row">
-          <div class="col d-flex justify-content-center gap-4 mt-2">
-            <a class="navbar-brand" href="#"
-              ><i class="bx bxs-bowl-hot">HealthyFood</i></a
-            >
-            <p>2023 World Food Conference</p>
+          <div class="col-6">
+            <img src="./img/logo.svg" alt="Healthy Food Logo" />
+          </div>
+          <div class="col-6 align-items-center d-flex justify-content-end">
+            <p class="m-0 text-right">2023 World Food Conference</p>
```

Comment:
- [ ] Please, notice that the partner section should not be present on the About page and the footer sections should be dark. Please remove the partner section from the About page and update the footer to dark with white text color:

**Your project about page:**
![image](https://github.com/PabloBona/Portal-Capstone-Project/assets/74411135/d1186c35-e704-4440-b240-5e204050db75)

**provided design about page**
![image](https://github.com/PabloBona/Portal-Capstone-Project/assets/74411135/624f39e0-c567-4499-b198-cf810fefff4e)

This will make your project prettier and make the user know you're an awesome developer as you can precisely follow a professional web developer's design 😃 💪